pkg/apps/installer.go

```diff
@@ -121,6 +121,23 @@ func (i *Installer) Update() {
 	return
 }
 
+// Delete will remove the application linked to the installer.
+func (i *Installer) Delete() (*Manifest, error) {
+	if i.man == nil {
+		return nil, ErrNotFound
+	}
+	if state := i.man.State; state != Ready && state != Errored {
+		return nil, ErrBadState
+	}
+	if err := deleteManifest(i.ctx, i.man); err != nil {
+		return nil, err
+	}
+	if err := vfs.RemoveAll(i.ctx, i.appDir()); err != nil {
+		return nil, err
+	}
+	return i.man, nil
```

Comment:
IIRC, we said that we have to remove some permissions when uninstalling an app. Probably the reverse of `permissions.CreateAppSet`